src/IssueViz/Editor/SelectedIssueTagging/ThemeColors.cs

```diff
@@ -0,0 +1,70 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Windows.Media;
+using Microsoft.VisualStudio.PlatformUI;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Editor.SelectedIssueTagging
+{
+    internal static class ThemeColors
+    {
+        internal static readonly IThemeColors LightTheme = new LightThemeColors();
+        internal static readonly IThemeColors DarkTheme = new DarkThemeColors();
+
+        public static bool IsLightTheme(Brush textBrush)
+        {
+            var isLightTheme = true;
+
+            if (textBrush is SolidColorBrush solidColorBrush)
+            {
+                var isDarkText = solidColorBrush.Color.IsDark();
+                isLightTheme = isDarkText;
+            }
+
+            return isLightTheme;
+        }
+
+        public static IThemeColors BasedOnText(Brush textBrush)
+        {
+            return IsLightTheme(textBrush) ? LightTheme : DarkTheme;
+        }
+    }
+
+    internal class LightThemeColors : IThemeColors
```

Comment:
I went with colors that are brighter than the ones we use in other SL flavors.
Which ones do you think look better?
Top is the ones in this PR, bottom is the ones in other SL flavors.

![image](https://user-images.githubusercontent.com/60586879/93365425-1150d580-f84a-11ea-9cae-9e8f70a0f7dc.png)

Reply:
My preferences would be top-right and bottom-left, but it's no a strong preference.
The light border around the highlight in dark mode looks a bit odd, though.